modules/library/src/main/scala/zio/elasticsearch/ElasticRequest.scala

```diff
@@ -656,8 +656,8 @@ object ElasticRequest {
     def searchAfter(value: Json): SearchAndAggregateRequest =
       self.copy(searchAfter = Some(value))
 
-    def sort(sorts: Sort*): SearchAndAggregateRequest =
-      self.copy(sortBy = sortBy ++ sorts.toSet)
+    def sort(sort: Sort, sorts: Sort*): SearchAndAggregateRequest =
+      self.copy(sortBy = (sortBy :+ sort) ++ sorts.toList)
```

Comment:
If we change (sortBy :+ sort) ++ sorts.toList to sortBy ++ (sort :: sorts.toList) then sortBy only gets copied once instead of twice.

On a separate note: should we use Chunk instead?